build.sbt

```diff
@@ -628,6 +630,58 @@ lazy val sbtProj = (project in file("sbt"))
   )
   .configure(addSbtCompilerBridge)
 
+lazy val sbtBig = (project in file(".big"))
+  .dependsOn(sbtProj)
+  .settings(
+    name := "sbt-big",
+    normalizedName := "sbt-big",
+    crossPaths := false,
+    assemblyShadeRules.in(assembly) := {
+      val packagesToBeShaded = Seq(
+        "com",
+        "fastparse",
+        "gigahorse",
+        "google",
+        "jawn",
+        "jline",
+        "native",
+        "okhttp3",
+        "okio",
+        "org",
+        "sbinary",
+        "sbt",
+        "scala",
+        "scalacache",
+        "scalapb",
+        "sjsonnew",
+        "sourcecode",
+        "xsbt",
+        "xsbti",
```

Comment:
My concern about this PR is this shading list.
Shading of anything `sbt.` or anything intrinsic to sbt doesn't make sense.
Next, the question would be where we would draw the line for third-party libraries.

Reply:
I agree sbt-intrinsics and scala-library should not be shaded. I would shade packages that have breaking changes since sbt 1.0, the ones I'm aware of are

- fastparse
- scalapb
- jawn

More packages can be shaded in the future in case problems arise.